src/variable.jl

```diff
@@ -56,6 +56,13 @@ Returns the name of a variable.
 """
 function name end
 
+"""
+    name_base(v::AbstractVariable)::Tuple{String, Vector{Int}}
```

Comment:
The name here is `name_base` instead of `name_base_indices`. In TypedPolynomials, it will most probably rather be a tuple than a `Vector{Int}`